AsyncDisplayKit/Layout/ASLayout.h

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <UIKit/UIKit.h>
+#import <AsyncDisplayKit/ASAssert.h>
+#import <AsyncDisplayKit/ASLayoutable.h>
+
+/** Represents the computed size of a layout node, as well as the computed sizes and positions of its children. */
+@interface ASLayout : NSObject
+
+@property (nonatomic, readonly) id<ASLayoutable> layoutableObject;
```

Comment:
This is really cool!  I am pretty happy with how this should be able to eliminate the ASCompositeNode.